actions.yaml

```diff
@@ -55,3 +55,26 @@ new-app-role:
       description: CIDR to restrict this app role to
       typoe: string
       default: '0.0.0.0/0'
+
+generate-certificate:
+  description: Generate a certificate against the Vault PKI.
+  params:
+    cn:
+      type: string
+      description: >-
+        CN (Common Name) field of the new certificate
+    sans:
```

Comment:
Fails while loading sans here [1]
The string requires conversion to a list and dump to json format

$ juju run -m openstack vault/0 generate-certificate cn=test sans="192.168.29.186 192.168.122.1 172.17.0.1 10.198.131.1 10.1.107.192" type=client 
Running operation 31 with 1 task
  - task 32 on unit-vault-0

Waiting for task 32...
ERROR the following task failed:
 - id "32" with return code 1

Exception:
  File "./src/charm.py", line 216, in issue_certificate
    sans = json.loads(sans)
  File "/usr/lib/python3.8/json/__init__.py", line 357, in loads
    return _default_decoder.decode(s)
  File "/usr/lib/python3.8/json/decoder.py", line 340, in decode
    raise JSONDecodeError("Extra data", s, end)

[1] https://github.com/ChrisMacNaughton/charm-vault-k8s/blob/master/src/charm.py#L208